src/components/ui/Breadcrumb/breadcrumb.scss

```diff
@@ -78,7 +78,7 @@
       }
     }
 
-    [data-store-dropdown-button] {
+    [data-fs-dropdown-button] {
```

Comment:
Looks like we didn't do this for NextJS, so maybe it's something we should not forget to add there later.

https://github.com/vtex-sites/nextjs.store/blob/2460a62cdae585628aee95b3b92540fd0106b54a/src/components/ui/Breadcrumb/breadcrumb.scss#L81

Reply:
I've noticed that I missed some details on NextJS version while doing this one 😓 so I've draft a PR to sync them https://github.com/vtex-sites/nextjs.store/pull/123

Thanks for the heads up :)